### DISCUSSION TITLE
Map Pages to App Guide

### DISCUSSION THREAD

**Discussion by benhaynes** [directus/directus#9242] [Feature Requests]
I'd like to create a mapping from each page of the App to relevant guides within the Docs App Guide. Once we have this, we can show `Relevant Guides` in the page's info sidebar. For example:

**Item Detail:**
* Creating an item
* Browsing revisions
* Archiving an item

**Comment by br41nslug** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
